module/VuFind/src/VuFind/Role/PermissionProvider/PluginManager.php

```diff
@@ -49,7 +49,8 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'insecureCookie' => InsecureCookie::class,
         'ipRange' => IpRange::class,
         'ipRegEx' => IpRegEx::class,
-        'role' => Role::class,
+    'role' => Role::class,
```

Comment:
The indentation looks weird here. Is it possible your editor is using tabs instead of spaces somewhere?

Reply:
thanks @demiankatz that is fixed in my follow-up